.github/workflows/main.yml

```diff
@@ -10,3 +10,18 @@ jobs:
       - uses: actions/checkout@v2
       - run: g++ -std=c++17 *.cpp
       - run: ./run_test.sh
+      - name: Install tools
+        run: sudo apt-get install -y valgrind cppcheck
+      - name: Gplusplus Wall
+        run: g++ -Wall -Werror *.cpp
+      - name: Valgrind
+        run: valgrind --error-exitcode=1 ./a.out units/ironman.json units/hulk.json
+      - name: Cppchecks
+        run: cppcheck *.cpp --enable=warning --error-exitcode=1
+      - name: Cppcheck report
+        run: cppcheck *.cpp --enable=warning --output-file=report.txt
+      - name: Artifacting
```

Comment:
Itt már a style és performance szintű hibákat kellene vizsgálni és artifactolni. Ha volt warning, akkor az előző lépésnél úgyis leáll a workflow.

Reply:
@hegrobaa Ez még nem lett javítva. Még mindig csak a warning szintű hibákat próbálja artifactolni, de ha lenne olyan, akkor el sem jut idáig.

Reply:
@oliverosz itt felcseréltem a kettőt, tehát az Artifacting rész megy le előbb (warning,style,performance szinten) és utána a warningokat csekkolja. Tehát így gondolom, az artifacting végrehajtódik és utána áll le a workflow warning esetén. Jól csináltam?

Reply:
Ok, így már jó. A lényeg az volt, hogy itt a style+performance szintek szerepeljenek, nem a sorrend, de talán jobb is ebben a sorrendben.